content/cybersecurity/concepts/nmap/terms/tcp-syn-scan/tcp-syn-scan.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'TCP SYN Scan'
+Description: 'Performs a scan quickly, efficientily and stealthly. The most popular scan option'
+Subjects:
+  - 'Networking'
+  - 'Cybersecurity'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'introduction-to-ethical-hacking'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **TCP SYN scan** runs by default when running Nmap as root or Administrator. It is the most popular scan option.
+Quick and efficient, this scan can indicate open, filtered and closed port states.
+It is requested by passing the -sS option to Nmap, but when running as root or Administrator, -sS is usually omitted.
+TCP SYN Scan is also stealthy because it never completes the full TCP connection, so is less likely to be blocked by firewalls.
+
+## Syntax
+
+To perform a TCP SYN scan on `<target>`, root privileges are required.
+Type the following command and enter user password when prompted:
```

Comment:
Can we move these to the same line? In addition, let's change the wording here to be informative rather than directing a user to do something and include what `<target>` stands for.

Reply:
```suggestion
Root privileges are required to perform a TCP SYN scan. The following command performs a scan on `<target>` after entering the user password when prompted:
```

Could the target description be included on the **Note** below @yangc95 ?